<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>661</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="3">
      <widget class="QPushButton" name="mainLocationBrowseButton">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLineEdit" name="mainRoot">
       <property name="statusTip">
        <string extracomment="test tip"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>/root</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="episodeLocation">
       <property name="statusTip">
        <string extracomment="test tip"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>/model/location.txt</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="episodeBrowseButton">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="mainLable">
       <property name="text">
        <string>Main Structure</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="episodeNumberLabel">
       <property name="text">
        <string>001</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="seasonLable">
       <property name="text">
        <string>Season</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="dirModelLocationLabel">
       <property name="text">
        <string>Directory model location</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QPushButton" name="mainGenerateButton">
       <property name="text">
        <string>Generate</string>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QPushButton" name="episodeGenerateButton">
       <property name="text">
        <string>Generate</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="episodeLable">
       <property name="text">
        <string>Episode</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="mainNumberLabel">
       <property name="text">
        <string>001</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="seasonNumberLabel">
       <property name="text">
        <string>001</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="seasonLocation">
       <property name="statusTip">
        <string extracomment="test tip"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>/model/location.txt</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="mainLocation">
       <property name="statusTip">
        <string extracomment="test tip"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>/model/location.txt</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="seasonBrowseButton">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item row="3" column="6">
      <widget class="QPushButton" name="seasonGenerateButton">
       <property name="text">
        <string>Generate</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QPushButton" name="mainRootBrowseButton">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="rootLocationLabel">
       <property name="text">
        <string>Root location</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>481</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="1">
      <widget class="QLineEdit" name="projectNameText">
       <property name="text">
        <string>Story Medicine</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="projectNameLabel">
       <property name="text">
        <string>Project Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="projectLoadButton">
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="projectSaveButton">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>783</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Generate</string>
    </property>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuCreate_Model">
    <property name="title">
     <string>Create Model</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCreate_Model"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>episodeBrowseButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>signalEpisodeBrowse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>374</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>episodeGenerateButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>signalEpisodeGenerate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>455</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mainGenerateButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>signalMainGenerate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>455</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mainLocationBrowseButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>signalMainBrowse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>374</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>seasonBrowseButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>signalSeasonBrowse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>374</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>seasonGenerateButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>signalSeasonGenerate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>455</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>projectLoadButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>signalLoadProject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>443</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>projectSaveButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>signalSaveProject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>343</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mainRootBrowseButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>signalMainRootBrowse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>523</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>391</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>signalEpisodeBrowse()</slot>
  <slot>signalEpisodeGenerate()</slot>
  <slot>signalMainBrowse()</slot>
  <slot>signalMainGenerate()</slot>
  <slot>signalSeasonBrowse()</slot>
  <slot>signalSeasonGenerate()</slot>
  <slot>signalSaveProject()</slot>
  <slot>signalLoadProject()</slot>
  <slot>signalMainRootBrowse()</slot>
 </slots>
</ui>
